dask_image/ndmeasure/__init__.py

```diff
@@ -29,7 +29,7 @@
     "minimum",
     "minimum_position",
     "standard_deviation",
-    "sum",
```

Comment:
I think we will need to keep both `sum` and `sum_labels` in this list for now. We want people to still be able to use `sum`, but they will also see a warning to change their code to use `sum_labels` instead. Later on, we will remove it entirely.

Reply:
+1